Seed.Tests/CharacterTests.cs

```diff
@@ -0,0 +1,49 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Text;
+using Seed.Characters;
+using NUnit.Framework;
+using FluentAssertions;
+using Seed.Items;
+using Seed.Locations;
+
+namespace Seed.Tests
+{
+    [TestFixture]
+    public class CharacterTests
+    {
+        [Test]
+        [Category("Character.ReceiveItems")]
+        public void ShouldHaveDamageAndArmorEqualToProperValues()
+        {
+            var location = new Location();
+            var human = new Human(presentLocation: location, strength: 2, armor: 1);
```

Comment:
currentLocation?

Reply:
I think it's unnecessary, because both "current" and "present" words in this case mean the same.
I found this article:
https://ell.stackexchange.com/questions/78020/difference-between-current-and-present
and it stands by my side :P